### PR TITLE
Link EF Core metrics documentation

### PR DESCRIPTION
**Pull request by cincuranet**
After https://github.com/dotnet/EntityFramework.Docs/pull/4743 is merged, this can be merged.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/diagnostics/built-in-metrics.md](https://github.com/dotnet/docs/blob/031010d558deefe1f6a6287580e395bae5f1632e/docs/core/diagnostics/built-in-metrics.md) | [Built-in metrics in .NET](https://review.learn.microsoft.com/en-us/dotnet/core/diagnostics/built-in-metrics?branch=pr-en-us-41361) |

### CODE REVIEW

**Review by roji — APPROVED** [review]

File: docs/core/diagnostics/built-in-metrics.md [hunk 1]
```diff
@@ -14,3 +14,4 @@ see the [EventCounters reference](available-counters.md) and [Windows Performanc
 - [.NET extensions metrics](built-in-metrics-diagnostics.md)
 - [ASP.NET Core Metrics](built-in-metrics-aspnetcore.md)
 - [System.Net Metrics](built-in-metrics-system-net.md)
+- [Entity Framework Core Metrics](../../../ef/core/logging-events-diagnostics/metrics)
```
Comment (adegeo) [inline docs/core/diagnostics/built-in-metrics.md:17]: This should fix the link
```suggestion
- [Entity Framework Core Metrics](/ef/core/logging-events-diagnostics/metrics)
```

**Review by adegeo — CHANGES_REQUESTED** [review]

**Comment by cincuranet** [comment]
@dotnet-policy-service rerun

File: docs/core/diagnostics/built-in-metrics.md (hunk 1 above)
Comment (cincuranet) [reply docs/core/diagnostics/built-in-metrics.md]: Thanks. I didn't realize, I'm "above" the "current" folder structure.

**Comment by adegeo** [comment]
Unsure why the build still says the page link is missing. I verified that it is published.

**Review by adegeo — APPROVED** [review]

**Pull request merged**